.github/workflows/snyk_scan.yaml

```diff
@@ -16,12 +13,19 @@ jobs:
         fetch-depth: 0
```

Comment:
this `depth` should be `1`, not `0`, I messed up the original documentation

Reply:
I may be misremembering, but I thought something in the `vanagon dependencies` command used a git ref like `HEAD` and that isn't available when using `fetch-depth: 1`. I'll switch it back to 0 to see what happens.